builds/misc/templates/base-image-publish/base-image-overview.yaml

```diff
@@ -0,0 +1,26 @@
+steps:
+- pwsh: |
+    # Note: With in the same job, you can access $(info), but it would interpret $(info) as string similar to `echo "$info"` which is undesirable for our case, we want an Object.
+    # We worked around this by using the serialization & deserailzation
+    $info = '$(serialize_info)' | ConvertFrom-Json
+
+    # Fetch tags & version from each module Dockerfiles {../$(arch)/base/Dockerfile, ../$(arch)/Dockerfile}
+    foreach ($each in $info)
```

Comment:
`$eachInfo` ? lol.

Reply:
I think something that indicates that this info is specific to one base image would help people unfamiliar with the whole process. Maybe baseImageContext?

Reply:
Changed to `config`
For more explicit & convention on the procedure, I'll defer to a user manual (which is to be written).

Reply:
nit: Still think we could make it more clear that this config is specific to one base image. The user manual makes sense, but I think the code can still be more self-documenting.

Reply:
I see where you're coming from. Without a paragraph explaining that each "base image" has an underlying OS base image associate with them. I'm not sure how else I can explain it. The user should read the code to see what it is doing. Otherwise, you will run into the maintenance hassle between (code comment vs code vs document). I drew the line that anything conceptual & user manual are going to be explained in document. For more detailed implementation, let the user read the code. Naming convention can help so much when a user doesn't have an understand of what we are trying to do.